.github/pull_request_template.md

```diff
@@ -36,3 +36,38 @@ _Please delete options that are not relevant._
 - [ ] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
 - [ ] My changes generate no new warnings
 - [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.
```

Comment:
Add a new check point for this:
- I have added the project meta data in the PR template.